QUICKSTART.md

```diff
@@ -225,6 +225,36 @@ public void handle(HttpExchange he) {
 
 ## Metrics
 
+Spans are a great way to get detailed information about what your application is doing, but
+what about a more aggregated perspective? OpenTelemetry provides supports for metrics, a time series
+of numbers that might express things such as CPU utilization, request count for an HTTP server, or a
+business metric such as transactions.
+
+All metrics can be annotated with labels: additional qualifiers that help describe what
+subdivision of the measurements the metric represents.
+
+The following is an example of metric usage:
+
+```java
+// Gets or creates a named meter instance
+Meter meter = OpenTelemetry.getMeterProvider().get("library_name");
+
+// Build counter e.g. LongCounter 
+LongCounter counter = meter
+        .longCounterBuilder("processed_jobs")
+        .setDescription("Processed jobs")
+        .setUnit("1")
+        .setLabelKeys(Collections.singletonList("Key"))
+        .build();
+
+// It is recommended that the API user keep a reference to a Bound Counter
+BoundLongCounter someWorkBound = counter.bind("SomeWork");
+
+// Record data
+someWorkBound.add(123);
```

Comment:
```suggestion
BoundLongCounter someWorkCounter = counter.bind("Key", "SomeWork");

// Record data
someWorkCounter .add(123);
```

`bind` needs to be supplied with a list of key/value pairs, not just the values.
Also the use of "Bound" as a subject name was dropped a while ago.

Reply:
updated

Reply:
The proper code woudl be:  
```java
BoundLongCounter someWorkBound = counter.bind(meter.createLabelSet("Key", "SomeWork"));
```

Reply:
method `meter.createLabelSet(...)` doesn't exist

Reply:
@thisthat not anymore since LabelSet was removed in #992

Reply:
Ops! I have to get up2date with the Metric API then! :sweat_smile: